go.mod

```diff
@@ -36,7 +36,7 @@ require (
 require (
 	github.com/natefinch/atomic v1.0.1
 	k8s.io/cloud-provider v0.26.2
-	k8s.io/cloud-provider-gcp/crd v0.0.0-20230202183644-b674bb5be613
+	k8s.io/cloud-provider-gcp/crd v0.0.0-20230315181918-038d2342e967
```

Comment:
Where do you get this value from? This is what tripped me up in my last CL.

Reply:
We should also make it such that we do not need to update go.mod and go.sum if we make changes to the CRD directory. We could be making a change to crd just for Anthos. If we have to update this here, then the change applies to GKE as well. We would ideally want to decouple these changes.